### PR TITLE
Add random id category

### PR DESCRIPTION
**Pull request by kreztoffee**
This adds a new category that will generate a random string containing chars and digits as defined by the regex type expression.

Library for this string generation can be found here: https://pypi.org/project/StringGenerator/

Let me know if you're happy with the notation and implementation. Otherwise I think this adds a nice addition for random 'uniqueness' to generated qual-ids

### CODE REVIEW

**Comment by kreztoffee** [comment]
I will address code coverage soon, didn't spot this!

File: qual_id/pattern.py [hunk 1]
```diff
@@ -87,6 +88,7 @@ class Pattern:
         "pokemon": Pokemon(),
         "profession": Profession(),
         "programminglanguage": ProgrammingLanguage(),
+        "randomid": RandomId(),
```
Comment (gabrielbarker) [inline qual_id/pattern.py:91]:
```suggestion
        "id": RandomId(),
```
I'd be happy with just 'id' as the url parameter, but feel free to disagree 🙂

**Review by gabrielbarker — COMMENTED** [review]
Thanks so much @kreztoffee ! This looks great
Regarding coverage, don't feel you have to go really thorough on the testing (mocking StringGenerator etc.), I'd be happy with a simple length check or even a regex pattern, totally up to you

File: qual_id/pattern.py (hunk 1 above)
Comment (kreztoffee) [reply qual_id/pattern.py]: I have made this change. I kept the class name the same as I didn't want to step on anything system like.... maybe not an issue

**Review by gabrielbarker — APPROVED** [review]
Thanks so much @kreztoffee

**Pull request merged**